pkg/sync/sync.go

```diff
@@ -681,12 +681,12 @@ func startSingleProducer(tasks chan<- object.Object, src, dst object.ObjectStora
 	start, end := config.Start, config.End
 	logger.Debugf("maxResults: %d, defaultPartSize: %d, maxBlock: %d", maxResults, defaultPartSize, maxBlock)
 
-	srckeys, err := ListAll(src, prefix, start, end, !config.Links)
```

Comment:
The list interface should follows links by default

Reply:
reverted